jekyll/_cci2/configuration-reference.md

```diff
@@ -459,19 +459,14 @@ jobs:
 
 ##### Machine Executor (Linux)
 
-Class            | vCPUs | RAM
------------------|-------|------
-medium (default) | 2     | 7.5GB
-large            | 4     | 15GB
-xlarge           | 8     | 32GB
-2xlarge          | 16    | 64GB
-{: class="table table-striped"}
+{% include snippets/machine-resource-table.md %}
 
 ###### Example Usage
 ```yaml
 jobs:
   build:
-    machine: true
+    machine:
```

Comment:
Just stuck this change in as spotted it was inconsistent with other examples. This Ubuntu image is, for now, the recommended image to use, rather than the default.